packages/grid/veilid/server/veilid_core.py

```diff
@@ -151,7 +153,11 @@ async def app_call(vld_key: str, message: bytes) -> bytes:
         async with await get_routing_context(conn) as router:
             route = await get_route_from_vld_key(vld_key, conn, router)
 
-            result = await router.app_call(route, message)
+            result = (
+                await router.app_call(route, message)
```

Comment:
I think , we might also need to handle the case, when the request could be less than 32 KB, but the response might be a large value.